rust/pact_verifier_cli/Cargo.toml

```diff
@@ -49,3 +49,11 @@ expectest = "0.12.0"
 rstest = "0.24.0"
 tempfile = "3.20.0"
 trycmd = "0.15.9"
+
+
+[[bin]]
+name = "pact-verifier"
+path = "src/bin.rs"
```

Comment:
More conventionally, you would have the file be `/src/bin/pact-verifier.rs`, and then I think you can drop the custom path.